eth/tracers/api.go

```diff
@@ -283,7 +283,14 @@ func (api *API) traceChain(ctx context.Context, start, end *types.Block, config
 						TxIndex:   i,
 						TxHash:    tx.Hash(),
 					}
-					res, err := api.traceTx(localctx, msg, txctx, blockCtx, task.statedb, config)
+
+					l1DataFee, _, _, err := fees.CalculateFees(tx, task.statedb)
+					if err != nil {
+						log.Warn("CalculateFees failed", "hash", tx.Hash(), "block", task.block.NumberU64(), "err", err)
+						break
```

Comment:
Why not return error here?

Reply:
fixed https://github.com/scroll-tech/go-ethereum/pull/351/commits/71b775d03efd523f32b9d543be379f4b66b1cbe2